<launch>
 
  <arg name="local_robot_id" default="0" />
  <arg name="logs_location" default="/root/rdpgo_ws/src/robust_distributed_slam_module/scripts/log/"/>
  
  <arg name="fcu_port" default="/dev/ttyTHS2" />
  <arg name="baud_rate" default="115200" />

  <group ns="robot_$(arg local_robot_id)">
    <include file="$(find dji_sdk)/launch/sdk_manifold.launch">
       <arg name="fcu_port" value="$(arg fcu_port)" />
       <arg name="baud_rate" value="$(arg baud_rate)" /> 
    </include>
  </group>


  <node pkg="rosbag" type="record" name="rosbag_record_log_pos"
       		args="record -o $(arg logs_location)camera_logs -e /robot_$(arg local_robot_id)/dji_sdk/(.*) /robot_$(arg local_robot_id)/camera/color/camera_info /robot_$(arg local_robot_id)/camera/color/image_raw /robot_$(arg local_robot_id)/camera/infra1/camera_info /robot_$(arg local_robot_id)/camera/infra1/image_rect_raw /robot_$(arg local_robot_id)/camera/infra2/camera_info /robot_$(arg local_robot_id)/camera/infra2/image_rect_raw /tf /tf_static"/>

</launch>
